### PR TITLE
Add issue template for enhancements

### PR DESCRIPTION
**Pull request by jlowin**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-feature-request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: New feature request
```
Comment (cicdw) [inline .github/ISSUE_TEMPLATE/new-feature-request.md]: We already have a feature request template, right?

Comment (jlowin) [reply .github/ISSUE_TEMPLATE/new-feature-request.md]: This is surprising, I renamed the template and GitHub didn't erase the old one. To be clear, I don't write these by hand -- they're all generated by a GitHub tool. Surprised it doesn't handle renames...

Comment (jlowin) [reply .github/ISSUE_TEMPLATE/new-feature-request.md]: updated by hand

**Pull request merged**